streams/src/main/java/org/apache/kafka/streams/StreamsBuilder.java

```diff
@@ -222,9 +222,10 @@
         Objects.requireNonNull(topic, "topic can't be null");
         Objects.requireNonNull(consumed, "consumed can't be null");
         Objects.requireNonNull(materialized, "materialized can't be null");
-        materialized.withKeySerde(consumed.keySerde).withValueSerde(consumed.valueSerde);
+        final ConsumedInternal<K, V> consumedInternal = new ConsumedInternal<>(consumed);
+        materialized.withKeySerde(consumedInternal.keySerde()).withValueSerde(consumedInternal.valueSerde());
```

Comment:
Needed to make these changes as Consumed moved packages and does not have access to key and value serde same for below in `StreamsBuilder` class.